styleguide/components/41-organisms/footer/_footer.scss

```diff
@@ -57,13 +52,13 @@
         width: 100%;
 
         @include phablet {
-          order: 3;
-          width: span(6 of 12);
+          width: span(12 of 12);
```

Comment:
Why change the width?

Reply:
The first `.footer--column` is the gentinfo block; it should be displayed over the full width of the footer. Previously, it was displayed on the last place, so the 100% width was implied because of flexbox. But now that the order has changed (and the gentinfo block is first in order), the width needs to be set explicitly in order to reach 100% width. 

Does that make sense? 🤔

Reply:
Yup :) makes sense.